### ISSUE TITLE
[WEB-429] Colour contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by nathanbillis** [YUSU-Dev/website#346]
There is insufficient colour contrast between the text and background on this page to meet the minimum contrast requirement.

For the suggestions/view page the following are too low:

Covid-19

Contrast ratio = 4:1

Fast Tracked

Contrast ratio = 2.28:1

<sub>From [SyncLinear.com](https://synclinear.com) | [WEB-429](https://linear.app/yorksu/issue/WEB-429/colour-contrast-does-not-meet-minimum-requirement)</sub>
